README.md

```diff
@@ -200,7 +208,7 @@ BitShares UI bugs should be reported to the [UI issue tracker](https://github.co
 Up to date online Doxygen documentation can be found at [Doxygen.BitShares.org](https://doxygen.bitshares.org/hierarchy.html).
 
 
-Using the API
+Built-In APIs
```

Comment:
This will break the index at the beginning of this document. Please update that too.